### ISSUE TITLE
A11Y: Resize handle does not have correct `role`.

### ISSUE THREAD

**Issue by BetaZhang** [tinymce/tinymce#9283]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](fiddle.tiny.cloud) or similar.
1. Use axe DevTools scan the page.

### ✔️ Expected result
*What is the expected result of the above steps?*
1. Resize handle should have correct role.

### ❌ Actual result
*What is the actual result of the above steps?*
1. Resize handle does not have correct role.
![WXWorkCapture_17044410684728](https://github.com/tinymce/tinymce/assets/102902468/2920acff-521a-4cc9-8a20-352ec7bcff33)

### ❓ Possible solution
*If you have ideas, you can list them here. Otherwise, you can delete this section.*
Maybe the role should be `"button"`.

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by lostkeys** [comment]
There exist no good role for the resize handle. Previous discussions suggest using the [separator role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role) for the time being. Curious to hear other peoples opinion on this.

https://github.com/w3c/aria/issues/1348
https://github.com/w3c/aria/issues/1443

— Fredrik, Product Manager
